src/frontend/shortlist/src/components/layout/signup_login/Loginview.vue

```diff
@@ -0,0 +1,126 @@
+<script>
+import Navbar from "./Navbar.vue";
+import Logo from "./Logo.vue";
+
+export default {
+  name: "Loginview",
+  components: { Navbar, Logo },
+  data() {
+    return {
+      // For log in
+      username_login: "",
+      password_login: "",
+      email_login: "",
```

Comment:
Do we require an email address for login? as we verify email in the signup process and then the user's account gets created with username and password.

Reply:
Yes.... by user-friendly convention. I understand what you said about no mentioning of email in js file to make things easier for Alex. But it would be comical if we don't require email for signup.